src/sql/src/plan/statement/show.rs

```diff
@@ -506,6 +506,22 @@ pub fn show_columns<'a>(
     ShowColumnsStatement { table_name, filter }: ShowColumnsStatement<Aug>,
 ) -> Result<ShowSelect<'a>, PlanError> {
     let entry = scx.get_item_by_resolved_name(&table_name)?;
+    let full_name = scx.catalog.resolve_full_name(entry.name());
```

Comment:
Not a big deal but it might be slightly more efficient to only resolve the name if we are bailing.

Reply:
Ah true. I'm going to cash in on my green CI build but will address in the follow up PR to make `SHOW COLUMNS` work for indexes too.